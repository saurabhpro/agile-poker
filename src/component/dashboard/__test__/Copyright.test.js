import React from 'react';
import { shallow, mount } from 'enzyme';
import Copyright from '../Copyright';

let component;

describe('copyright message', () => {
  beforeAll(() => {
    component = shallow(<Copyright />);
    //console.debug(component.debug());
  });

  it('matches the rendered snapshot', () => {
    expect(component).toMatchSnapshot();
  });

  it('should display name and current year', () => {
    expect(component.text()).toContain('Saurabh');
    expect(component.text()).toContain(new Date().getFullYear());
  });

  it('mounts', () => {
    component = mount(<Copyright />);

    //console.debug(component.debug());
    component.unmount();
  });
});
