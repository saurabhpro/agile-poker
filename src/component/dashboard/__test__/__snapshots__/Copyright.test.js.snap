// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`copyright message matches the rendered snapshot 1`] = `ShallowWrapper {}`;
